index.js

```diff
@@ -23,9 +24,9 @@ function createColormap (spec) {
         b = [],
         a = [];
 
-    if ( !at.isPlainObject(spec) ) spec = {};
+    if ( !isPlainObject(spec) ) spec = {};
```

Comment:
We could cut this dep as well. For example if a user wants to pass in an instance of a prototype as a specifcation object (TypeScript compat for example) they should be able to. If it walks like a duck...

A check for falsey values is probably sufficient.